BFE_RShiny/oasisui/R/exposurevalidation_map_module.R

```diff
@@ -628,13 +641,33 @@ exposurevalidationmap <- function(input,
 
   # Drawn circles infos, outputs and radius
   .DrawnCircles <- function(radius, lat_click, long_click, ratio) {
-
     # calculate LocID and TIV for pins inside areas
-    locID_list <- .is_within_bounds(format(result$uploaded_locs_check_peril$LocNumber, big.mark = "", scientific = FALSE), radius, lat_click, long_click, ratio)
-    tiv_list <- .is_within_bounds(result$uploaded_locs_check_peril$BuildingTIV * (ratio/100), radius, lat_click, long_click, ratio)
-    street_address <- .is_within_bounds(result$uploaded_locs_check_peril$StreetAddress, radius, lat_click, long_click, ratio)
+    if(is.null(result$uploaded_locs_check_peril$StreetAddress)) {
```

Comment:
make a two-column data.frame first. then if `StreetAddress` is not NULL, bind it to the data.frame as a third column.
afterwards call `.is_within_bounds` (without further condition).